windu.el

```diff
@@ -180,11 +189,28 @@
            (windu-swap-buffers other-window window)
            other-window))))
 
-;;; end-user interactive wrapper functions
+;;; window configuration hotkeys
+
+(defvar windu-window-configuration-2 nil)
+(defvar windu-window-configuration-3 nil)
+(defvar windu-window-configuration-4 nil)
+(defvar windu-window-configuration-5 nil)
+
+(defun windu-set-window-configuration(name)
+  "Save the current window configuration into name."
+  (set name (current-window-configuration))
+  (message "Window configuration saved"))
 
+(defun windu-load-window-configuration(name)
+  "Load the window configuration corresponding to name."
+  (if name (progn (set-window-configuration name) (message "Window configuration loaded"))
+    (message "Saved configuration not found")))
+
+;;; end-user interactive wrapper functions
 (defun windu-transient-activate ()
   "Begins a new nudge action."
   (interactive)
+  (message "Entering windu mode")
```

Comment:
This is a good idea.  What if we change it to `(windu-echo-size)` instead, so it prints the same message as when you're nudging things around?